server/game/cards/13.4-BtRK/ShadowOfTheIsles.js

```diff
@@ -0,0 +1,36 @@
+const DrawCard = require('../../drawcard');
+const GameActions = require('../../GameActions');
+const Messages = require('../../Messages');
+
+class ShadowOfTheIsles extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Choose and discard locations',
+            targets: {

```

Comment:
Since there's only a single *type* of target (non-limited location), this should use `target` (singular). Nothing Burns uses `targets` (plural) because it's selecting multiple types of targets. The keys nested under it are the target name ("attachment"), then the value is the target definition (prompt title, card condition, etc for choosing attachments).